### ISSUE TITLE
Awesome set of components but stuck due to docs inconsistency

### ISSUE THREAD

**Issue by kesirog** [Shopify/polaris-react-archive#8969]
First of all super awesome set of components, using v10.43.0 of Polaris.

I didn't bother to use components that are marked deprecated. 

Today there was a shift in the documentation:
* Columns(alpha) disappeared and were removed from docs
* New components appeared ~ Vertical stack, Horizontal stack, Horizontal grid, etc., (all alpha)

I'm writing this because I'm tight on time and inconsistency between latest documentation and latest Polaris version is stopping me from further development. (I'm aware of component lifecycle)

So my question is as of today, which components should I use to build basic layout structure since a lot of changes is going on very fast.

**Comment by kyledurand** [comment]
👋 Hey @kesirog it might be best to fork our components until they're in a more stable state, like beta or even mainstream. This would mean copying the tsx / scss files for whichever layout components you're using into your project. The only other alternative would be to stop upgrades on Polaris but that could lead to other issues as well

**Comment by kesirog** [comment]
@kyledurand Thank you for the super fast response! 🙌 I'm completely fine with locking the version. I was just confused because the docs included components that had not been released yet, and some components like Column suddenly disappeared. The reason I made this post was because I thought there was a faulty release.

So I stopped looking at the docs and instead read from the source code. It's not ideal, but at least I made some progress.
